### ISSUE TITLE
404 not found error

### ISSUE THREAD

**Issue by sofire** [perftools/xhgui#196]
http://xxx.com/run/view?id=58a28dbf6602cf14fc1119f2

In Nginx conf:
```
location / {
  try_files $uri $uri/ /index.php?$uri&$args;
}
```
QUERY_STRING in $_SERVER :
"/run/view&id=58a28dbf6602cf14fc1119f2"

Should be:
```
location / {
  try_files $uri $uri/ /index.php?$args;
}
```

**Comment by markstory** [comment]
Would you be willing to update the README with the correct nginx configuration>

**Comment by markstory** [comment]
Fixed in #197
